inspectors/types/pedaltypes.h

```diff
@@ -3,19 +3,18 @@
 
 #include "qobjectdefs.h"
 
-class PedalTypes
-{
+class PedalTypes {
     Q_GADGET
 
 public:
-    enum HookType {
+    enum class HookType : signed char {
```

Comment:
leave just enum class without type specification
Q_ENUM will convert it to int anyway and this may lead to the unexpected problems due to bit depth
We need to specify it only where it's necessary (like char_16t for UTF-16 unicodes)

Reply:
OK